requirements/constraints.txt

```diff
@@ -250,7 +250,7 @@ trustme==1.1.0 ; platform_machine != "i686"
     # via -r requirements/test.in
 typer==0.6.1
     # via python-on-whales
-typing-extensions==4.11.0
+typing-extensions==4.12.2
     # via
     #   aioredis
```

Comment:
That should be updated, aioredis is merged into redis now (and we probably want to move to valkey now).